specs/create-todo.spec.ts

```diff
@@ -0,0 +1,16 @@
+import { test, expect } from '@playwright/test';
+import { TodoPage } from '../pages/TodoPage';
+
+test('Add To Do', async({page})=>{
+    const todoPage = new TodoPage(page);
+
+    await todoPage.goto();
+
+    await todoPage.addToDoItem('title 1', 'description 1');
+
+    const ToDoItemList = await page.evaluate(()=> {
+        return JSON.parse(localStorage.getItem('todolist') || '[]')
+    }) 
+
+    expect(ToDoItemList[0].title).toBe('title 1')
```

Comment:
It's better to move hardcoded values into separate variable.

For example - `title 1` could be used in the another expectation or any action. It would be easier to change value in one place, instead of replacing all hardcoded values